op_crates/web/02_abort_signal.js

```diff
@@ -64,22 +64,35 @@
   }
 
   const handlerSymbol = Symbol("eventHandlers");
+
+  function makeWrappedHandler(handler) {
+    function wrappedHandler(...args) {
+      if (typeof wrappedHandler.handler !== "function") {
+        return;
+      }
```

Comment:
In what situation this branch would be true?

Reply:
In the test basically:
```js
ee.onabort = () => console.log('irrelevant');
ee.onabort = () => null;
ee.addEventListener('abort', e => console.log(1);
ee.onabort = () => console.log(2);
```

In this case, the target has to fire "1 2" and not "2 1" so we need to "keep the slot".

the DOM is weird like that 🤷

Reply:
The test tests this btw

Reply:
Thanks, that makes sense 👍